src/components/Todo/Todo.js

```diff
@@ -1,8 +1,17 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ListItem from 'components/ListItem/ListItem'
 
-const Todo = ({ children, ...props }) => {
-  return <ListItem {...props}>{children}</ListItem>
+const Todo = ({ icon, children, ...props }) => {
+  const [visible, setVisible] = useState(false)
```

Comment:
Para coordinarnos con Flor, vamos a cambiarlo por `isMouseOver` y `setIsMouseOver`. La razón de esto es que el nombre es más indicativo de lo que realmente está representando. De todas formas, si fuera una única cosa, el nombre está perfecto, pero a medida que vayamos agregando elementos que tengan que mostrarse, se va a ir quedando corto.

```suggestion
  const [isVisible, setIsVisible] = useState(false)
```

Reply:
Pablo no entendi, cambiamos por isMouseOver o por isVisible, porque el comentario dice una cosa y la sugerencia otra. Gracias

Reply:
Es verdad, mala mía, ahí lo modifiqué!